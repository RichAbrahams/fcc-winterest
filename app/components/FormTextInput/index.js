/**
*
* FormTextInput
*
*/

import React from 'react';
import { Input } from './styles';

function FormTextInput(props) {
  return (
    <Input>
      {props.children}
    </Input>
  );
}

FormTextInput.propTypes = {
  children: React.PropTypes.object,
};

export default FormTextInput;
