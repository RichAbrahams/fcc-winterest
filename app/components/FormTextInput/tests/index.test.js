import React from 'react';
import renderer from 'react-test-renderer';
import FormTextInput from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FormTextInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
