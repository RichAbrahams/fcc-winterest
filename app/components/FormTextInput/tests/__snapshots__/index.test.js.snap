exports[`test renders correctly 1`] = `
<input
  className="gVhbXr" />
`;
